app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -104,6 +106,12 @@ const FileInput = forwardRef((props, ref) => {
   const instanceId = useInstanceId();
   const { isMobile } = useContext(DeviceContext);
   const [isDraggingOver, setIsDraggingOver] = useState(false);
+  const hadValue = useRef(false);
```

Comment:
`value` can be initially set.

```suggestion
  const hadValue = useRef(Boolean(value));
```

Reply:
In adding a test case, I noticed this is already being tested. And in fact, it does work as expected, because `isUpdated`'s `useMemo` is called on the initial render and will immediately set `hadValue.current = true` if `value` is initially provided.

A `false` default could make sense because "had" in the sense of the initial render should be `false`, since there was no _previous_ state. 🤔